src/disabled-tasks/style.scss

```diff
@@ -0,0 +1,34 @@
+.free-trial-disabled-tasks {
+	li {
+		&:first-child {
+			grid-template-columns: 95% auto !important;
+			box-shadow: none !important;
+			p {
+				padding: 0px;
+			}
+		}
+	}
+
+	.woocommerce-experimental-list-wrapper {
+		border-top: none;
+	}
+
+	.free-trial-disabled-tasks-content {
+		font-size: 13px;
+		padding: 24px;
+		color: #1e1e1e;
+		p {
+			padding: 0;
+			margin: 0;
+		}
+		p.disabled-task {
+			background: #f0f0f080;
+			padding: 16px;
+			margin-bottom: 8px;
+			color: #949494;
```

Comment:
I created a follow-up task https://github.com/Automattic/wc-calypso-bridge/issues/941 to add `@wordpress/base-styles` support so that we can use `$gray-900` here.